src/bci_build/package.py

```diff
@@ -1838,7 +1838,8 @@ def _create_php_bci(
 {DOCKERFILE_RUN} chmod +x /usr/local/bin/docker-entrypoint.sh; \
     ln -s su /usr/bin/gosu; \
     mkdir /docker-entrypoint-initdb.d; \
-    sed -ri "s|^#?(listen_addresses)\s*=\s*\S+.*|\1 = '*'|" /usr/share/postgresql{ver}/postgresql.conf.sample
+    sed -ri "s|^#?(listen_addresses)\s*=\s*\S+.*|\1 = '*'|" /usr/share/postgresql{ver}/postgresql.conf.sample; \
+    mkdir -p "$PGDATA" && chown -R postgres:postgres "$PGDATA"
```

Comment:
Small nitpick, I would prefer

```
install -d -m 700 - owner postres $PGDATA

```

Here

Reply:
actually all of these lines are unnecessary if USER is set to postgres - the volume mounts are working then as expected.